app/screens/settings/notification_settings/notification_settings.js

```diff
@@ -188,6 +213,40 @@ class NotificationSettings extends PureComponent {
         }
     };
 
+    /**
+     * shouldSaveAutoResponder
+     *
+     * Necessary in order to properly update the notifyProps when
+     * enabling/disabling the Auto Responder.
+     *
+     * Reason being, on mobile when the AutoResponder is disabled on the server
```

Comment:
Is this something missing from the mobile app's websocket handling? Should we file a ticket to investigate this further?

Reply:
Yes, would be great to investigate what's missing on the mobile side

Reply:
@miguelespinoza I know it's been a while, so what is the actual behaviour that is a bug here? Is the mobile app not receiving the updated status or are the updated notify_props not being received?